### PR TITLE
Crystal Business Funding 

### PR DESCRIPTION
**Pull request by Crystal-Business-Funding**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Review by dhawton — CHANGES_REQUESTED** [review]
Remove the excess images, blank lines in `data/companies.yml`

**Comment by dhawton** [comment]
@Crystal-Business-Funding you have added 5 logos. Please remove the excess.

File: data/companies.yml [hunk 1]
```diff
@@ -602,7 +602,10 @@ users:
   - name: "GoTo Financial"
     logo: "/logos/gotofinancial.svg"
     url: "https://gotofinancial.com/"
-
+  - name: "Crystal Business Funding"
+    logo: "/logos/2lA8NvZqXk5hbjTObOtH37C4kAV.svg"
```
Comment (dhawton) [inline data/companies.yml:606]:
```suggestion
    logo: "/logos/2lA8NvZqXk5hbjTObOtH37C4kAV.svg"
```

Please name this similar to `CrystalBusinessFunding.svg` and remove the other images you are adding. The `2lA8N...` appears to be the desired photo, but the others need to be dropped and this renamed to something more understandable. Happy to approve once this is done.

Comment (Crystal-Business-Funding) [reply data/companies.yml]: I don't see the other photo's 
IMG-20240826-WA0035

Comment (dhawton) [reply data/companies.yml]: Screenshot_219

**Comment by craigbox** [comment]
I'm very sorry if this is off base, but this doesn't seem legit.  I wonder if this is just URL spam, especially given the financial nature of the URL. (Which appears to be hosted on Google Sites.)
Can you please give some information about your company and your use of Istio?

**Comment by Crystal-Business-Funding** [comment]
> I'm very sorry if this is off base, but this doesn't seem legit. I wonder if this is just URL spam, especially given the financial nature of the URL. (Which appears to be hosted on Google Sites.) Can you please give some information about your company and your use of Istio?
@craigbox 

We have a site in development, it will be ready soon,
We will be using Istio for account management, are company provides business loans to small businesses, we will be having the whole process online (in the client portal) that is why we will be using Istio, a company profile on Istio would be nice.

**Comment by craigbox** [comment]
Istio isn't used for account management. I'm going to close this, sorry. 
Please feel free to send a new PR when you have the details of a case study on how you used Istio.

**Comment by Crystal-Business-Funding** [comment]
Lol

**Pull request closed without merging**